samples/Samples.AspNetCoreMvc31/Properties/launchSettings.json

```diff
@@ -8,46 +8,81 @@
     }
   },
   "profiles": {
-    "IIS Express": {
+    "StartupHook IIS Express": {
       "commandName": "IISExpress",
-      "launchBrowser": false,
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ADDITIONAL_DEPS": "$(SolutionDir)bin\\tracer-home\\AdditionalDeps",
+        "DOTNET_SHARED_STORE": "$(SolutionDir)bin\\tracer-home\\store",
+        "DOTNET_STARTUP_HOOKS": "$(SolutionDir)bin\\tracer-home\\netcoreapp3.1\\OpenTelemetry.AutoInstrumentation.StartupHook.dll",
+        "OTEL_DOTNET_AUTO_CONSOLE_EXPORTER_ENABLED": "true",
+        "OTEL_DOTNET_AUTO_ENABLED_INSTRUMENTATIONS": "AspNet,HttpClient,SqlClient",
+        "OTEL_DOTNET_AUTO_INSTRUMENTATION_PLUGINS": "Samples.AspNetCoreMvc.OtelSdkPlugin, Samples.AspNetCoreMvc31, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
+        "OTEL_SERVICE_NAME": "StartupHook.IISExpress",
+        "OTEL_TRACES_EXPORTER": "jaeger"
```

Comment:
I would like to see OTLP here. Is this configuration needed for testing purposes?

Reply:
Good point. Not needed, let me see if Jaeger already supports OTLP, if not I can add a OTel collector to the docker image.